### PR TITLE
Faster to_interval(Gmpq)

### PR DESCRIPTION
**Pull request by mglisse**
## Summary of Changes

Make conversion to interval through MPFR3 faster. Keep old code for compatibility with centos with mpfr-2.2 or 2.4.

## Release Management

* Affected package(s): Number_types

### CODE REVIEW

**Comment by lrineau** [comment]
Only for `master` (4.12)?

File: Number_types/include/CGAL/double.h [hunk 1]
```diff
@@ -138,8 +138,7 @@ inline double sse2fabs(double a)
   __m128d temp = _mm_set1_pd(a);
   
   temp = _mm_and_pd(temp, absMask.m);
-  _mm_store_sd(&a, temp);
-  return a;
+  return _mm_cvtsd_f64 (temp);
```
Comment (lrineau) [inline Number_types/include/CGAL/double.h]: Is that change on purpose? It does not seem related to the description of the PR?

Comment (lrineau) [reply Number_types/include/CGAL/double.h]: Marc told me that it was on purpose.

**Comment by mglisse** [comment]
Yes, only master, it is a new optimization, it doesn't seem necessary to backport.
